### PR TITLE
Add documentation of fork

### PR DESCRIPTION
**Pull request by dingobar**
Add documentation of our usage of the upstream repository.

### CODE REVIEW

File: README-dfds-data.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+[the main readme](./README.md).
+
+## Using Grafana
+
+You must log in with the shared credentials. They are found
+[here](https://eu-west-1.console.aws.amazon.com/systems-manager/parameters/grafana_admin_credentials/description?region=eu-west-1).
+
+## Adding and deploying dashboards
+
+Dashboards should be created/modified in the UI, then saved as JSON and put in the dashboards folder
```
Comment (KPLauritzen) [inline README-dfds-data.md:19]: How do you save a dashboard as json?

File: README-dfds-data.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Grafana dashboards
```
Comment (KPLauritzen) [inline README-dfds-data.md:1]: Should we add a link to this README in the main README? Or will that create merge conflicts when pulling upstream changes?

**Comment by KPLauritzen** [comment]
@nybyhansen accidentally merged. Not sure how to handle that

File: README-dfds-data.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+fetch it.
+
+## Deploying Grafana
+
+Grafana has been deployed manually using the powershell script referenced in
+[the main readme](./README.md).
+
+## Using Grafana
+
+You must log in with the shared credentials. They are found
```
[the remark this answers is not in the document]
Comment (KPLauritzen) [reply README-dfds-data.md:14]: We should mention the name of the capability somewhere. Also, a link to the graphana website?

File: README-dfds-data.md (hunk 2 above)
Comment (dingobar) [reply README-dfds-data.md:1]: Afraid of merge conflicts if I do this

**Pull request merged**